chain/checkpointing/storage.go

```diff
@@ -0,0 +1,36 @@
+package checkpointing
+
+import (
+	"crypto/sha256"
+	"encoding/hex"
+	"fmt"
+	"os"
+
+	"github.com/minio/minio-go/v7"
+	"github.com/minio/minio-go/v7/pkg/credentials"
+)
+
+func StoreConfig() {
+	// Initialize minio client object.
+	minioClient, err := minio.New(S3_HOST, &minio.Options{
+		Creds:  credentials.NewStaticV4(ACCESS_KEY_ID, SECRET_ACCESS_KEY, ""),
+		Secure: false,
+	})
```

Comment:
As mentioned above, it'd be great in the future if we can gather these values from config.